SofaKernel/modules/SofaSimpleFem/SofaSimpleFem_test/scenes/TetrahedronDiffusionFEMForceField.scn

```diff
@@ -21,10 +25,10 @@
             <EulerImplicitSolver name="EulerExplicitSolver" firstOrder="1" tags="heat" rayleighStiffness="0.1" rayleighMass="0.1" />
             <CGLinearSolver name="CG" iterations="1000" tolerance="1.0e-10" threshold="1.0e-30" tags="heat" />
             <MechanicalObject template="Vec1d" position="@../../temperatureLoader.position"  name="gridTemperature" bbox="0 0 0 0 0 0" tags="heat" />
-            <TetrahedronDiffusionFEMForceField template="Vec1d" name="DiffusionForceField" constantDiffusionCoefficient="1.0" tagMechanics="geom" tags="heat"/>
-            <DiagonalMass template="Vec1d" name="Mass" massDensity="1.0" tags="heat"/>
+            <TetrahedronDiffusionFEMForceField template="Vec1d" name="DiffusionForceField" constantDiffusionCoefficient="1.0" tagMechanics="geom" tags="heat" topology="@../Container"/>
```

Comment:
well, with the help of explicit template + topology + getContext(), it will find its associated position container (aka MechanicalObject). I can set it explicitly if you want